seatunnel-core/seatunnel-flink-starter/pom.xml

```diff
@@ -103,6 +103,17 @@
             <artifactId>junit</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
we can't use MySQL in the complier step.

Reply:
@ruanwenjun could you give @zhangyuge1 some suggestions?

Reply:
But if I don't use mysql in the compiler step,it can not find jdbc driver.

Reply:
Yes, we cannot use mysql.

Reply:
> 

Hi would show me the e.g. clickhouse project code path?

Reply:
> Yes, we cannot use mysql, we can use other db to test the jdbc case e.g. clickhouse.

Hi would you show me the e.g. clickhouse cide path?